cf_units/_udunits2_parser/parser/udunits2Lexer.py

```diff
@@ -1,6 +1,6 @@
-# Generated from /home/ruth/git_stuff/cf-units/cf_units/_udunits2_parser/parser/udunits2Lexer.g4 by ANTLR 4.11.1
```

Comment:
Not sure we should be re-formatting this (since it is auto-generated)? Perhaps add an exception to the pre-commit config.

Reply:
I have skipped the line length rule for these files.